src/veh_interact.cpp

```diff
@@ -1553,7 +1553,7 @@ void veh_interact::display_stats()
         w[6] -= iw;
         const vpart_info &info = veh->parts[mostDamagedPart].info();
         vehicle_part part = veh->parts[mostDamagedPart];
-        int damagepercent = 100 * part.hp / info.durability;
+        int damagepercent = 100 * double( part.hp() ) / info.durability;
```

Comment:
The cast is excessive: multiplication has greater precedence than division and the result is `int` anyways.